app/helpers/cycle_timetable_helper.rb

```diff
@@ -17,6 +17,10 @@ def after_apply_deadline(year = CycleTimetable.current_year)
     CycleTimetable.apply_deadline(year) + 1.day
   end
 
+  def cancel_application_deadline(year = CycleTimetable.current_year)
+    CycleTimetable.reject_by_default(year) + 1.day
```

Comment:
Because the deadline is 11:59, the next day _is_ minutes after the deadline. 

I have adjusted the clock for the job to run at 1am (it was set for 11am). So the job will run at 61 minutes after the deadline.

Reply:
Perfect, sounds good, thanks. But the deadline should be changed to apply_deadline, correct?

Reply:
Sure, I don't think it matters.

Reply:
Sorry, I didn't understood why it doesn't matter? 
It matters because if we don't cancel after apply deadline candidates would be in limbo until the reject by default dates kicks. So the change from reject by default to apply deadline is important.